source/partials/_changelog.erb

```diff
@@ -4,6 +4,7 @@
 
 | Publication date | Update                                                                                                                                                                                                                                                                                                                                            |
 |------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Jan 13 2025      | Updates guidance <a href="/integrate-with-integration-environment/authenticate-your-user">"Authenticate your user"</a> to add information about configuring the max_age parameter.                                                                 |
```

Comment:
```suggestion
| Jan 13 2025      | Updates guidance <a href="/integrate-with-integration-environment/authenticate-your-user">"Authenticate your user"</a> to add information about using the `max_age` parameter.                                                             |
```

Reply:
would recommend you include a second sentence following this first one to say ......"to add information about validating the `max_age` parameter."